core/src/epicli/data/common/ansible/playbooks/roles/preflight/tasks/main.yml

```diff
@@ -25,3 +25,32 @@
     fail_msg: "Your operating system [{{ ansible_distribution  }} - {{ ansible_distribution_version }}] version and distribution doesn't meet system requirements.
               Distributions meetings requirements are: {{ message_supported_oeses }}"
     success_msg:  "Operating system [{{ ansible_distribution  }} - {{ ansible_distribution_version }}] version and distribution meet system requirements."
+
+- name: Check if PostgreSQL version needs to be migrated to version installed from Software Collections to installed from PostgreSQL repository
+  block:
+
+  - name: Check if PostgreSQL version has been installed from Software Collections
+    command:  rpm -q rh-postgresql10-postgresql-server
+    failed_when: no
+    changed_when: no
+    register: rpm_status
+
+  - name: Check if there is unexpected error during package testing
```

Comment:
This task seems to be not needed after changing `failed_when` option in the previous one.